health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/UserDownload.js

```
@@ -55,20 +55,23 @@ const UserDownload = () => {
         <div>
 
 
-            <Card>
+            <Card type="secondary" >
                 <HeaderComp title="DOWNLOAD_USER_DATA" styles={{ color: "black" }} />
                 <TextBlock body="DOWNLOAD_DESC" />
                     {data?.ResourceDetails &&
                         [...data?.ResourceDetails].reverse().map((item, index) => {
                             let fileName = item?.additionalDetails?.fileName || `FileNo${item?.processedFilestoreId?.slice(0, 4) || ''}`;
```

Biome

[error] This let declares a variable that is only assigned once.

'fileName' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)